solutions/bio1/index.tsx

```diff
@@ -38,5 +38,7 @@ export * from "./ModernArt";
 export * from "./MuTorere";
 export * from "./AnagramNumbers";
 export * from "./Juggluging";
+export * from "./DistinctPrimeFactorisation";
+export * from "./GoldbachConjecture";
 export * from "./NumberLadder";
-export * from "./DistinctPrimeFactorisation";
\ No newline at end of file
+export * from "./DistinctPrimeFactorisation";
```

Comment:
Is the `export * from "./DistinctPrimeFactorisation";` line duplicated or am I tripping?